src/Approximations/iterative_refinement.jl

```diff
@@ -35,60 +35,61 @@ Type that represents the polygonal approximation of a convex set.
 ### Fields
 
 - `S`           -- convex set
-- `approx_list` -- vector of local approximations
+- `approx_list` -- vector of local approximations that are not finished yet
+- `constraints` -- vector of linear constraints that are already finished
```

Comment:
Perhaps we should define what it means for an approximation to be "finished" in the docstring (or the docs).